### PR TITLE
Pass ReadOnlySequence via in

### PR DESCRIPTION
**Pull request by benaadams**
Its a chunky `readonly struct`

### CODE REVIEW

File: src/Kestrel.Core/Internal/Http/HttpParser.cs [hunk 1]
```diff
@@ -52,7 +50,7 @@ public unsafe bool ParseRequestLine(TRequestHandler handler, ReadOnlySequence<by
                 // No request line end
                 return false;
             }
-            else if (TryGetNewLine(ref buffer, out var found))
+            else if (TryGetNewLine(in buffer, out var found))
```
Comment (muratg) [inline src/Kestrel.Core/Internal/Http/HttpParser.cs]: Do we need `in` here? (I heard this would work without adding in on the call site.)

Comment (benaadams) [reply src/Kestrel.Core/Internal/Http/HttpParser.cs]: Nope, will remove

Comment (benaadams) [reply src/Kestrel.Core/Internal/Http/HttpParser.cs]: Done

**Comment by KrzysztofCwalina** [comment]
Ah, just saw this one. Disregard my comment in the other PR.

**Pull request merged**